packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -1230,13 +1220,39 @@ export class StripeHelper {
             )
           );
         }
-        plan = await this.expandResource(subscription.plan, PLAN_RESOURCE);
-        abbrevProduct = await this.expandAbbrevProductForPlan(plan);
-        break;
+
+        /** @type {Plan | undefined | null} */
```

Comment:
These are JS-doc style type annotations, we should be using proper TypeScript now, though I'm confused by why expandResource would not already have the right type here.